.travis.yml

```diff
@@ -9,26 +9,22 @@ matrix:
         - name: "js lint"
           language: node_js
           node_js:
-              - "node"
-              - "lts/*"
+              - 13
```

Comment:
13 was current, but it's not [an LTS](https://nodejs.org/en/about/releases/)? The `Dockerfile` is pinned to 10 LTS, so let's make this use 10 also.